### PR TITLE
Update react-markdown

### PR DESCRIPTION
**Pull request by vilsbole**
Following Snyk report on ReDoS vulnerability in trim@0.0.1
introduced through react-markdown@3.3.0 › remark-parse@5.0.0 › trim@0.0.1

ie. https://app.snyk.io/org/mycrypto/project/2e524bf4-52f1-49d7-bd2d-7447b489a04e/pr-check/d870f534-febf-4288-a5ab-9eeae4804b0f

### CODE REVIEW

File: src/components/TranslateMardown.test.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { screen, simpleRender } from 'test-utils';
+
+import TranslateMarkdown from './TranslateMarkdown';
+
+const renderComponent = ({ source }: { source: string }) => {
```
Comment (FrederikBolding) [inline src/components/TranslateMardown.test.tsx:7]: Fix test file name typo

File: src/components/TranslateMarkdown.tsx [hunk 2]
```diff
@@ -11,15 +11,15 @@ interface Props {
 export const TranslateMarkdown = ({ source }: Props) => {
   return (
     <Markdown
-      escapeHtml={true}
-      unwrapDisallowed={true}
-      allowedTypes={['link', 'emphasis', 'strong', 'code', 'root', 'inlineCode']}
+      disallowedTypes={['html']}
```
Comment (FrederikBolding) [inline src/components/TranslateMarkdown.tsx:14]: Do we not want to stick to the currently allowed types?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/components/TranslateMarkdown.tsx [hunk 3]
```diff
@@ -11,15 +11,15 @@ interface Props {
 export const TranslateMarkdown = ({ source }: Props) => {
   return (
     <Markdown
-      escapeHtml={true}
```
Comment (FrederikBolding) [inline src/components/TranslateMarkdown.tsx:14]: Why are these other props removed?

Comment (vilsbole) [reply src/components/TranslateMarkdown.tsx:14]: The api has changed. Now html is escaped by default.

File: src/components/TranslateMarkdown.tsx (hunk 2 above)
Comment (vilsbole) [reply src/components/TranslateMarkdown.tsx:14]: For un unknown reason none of our translation are rendered when using `allowedTypes` so we use the inverse just to ban what is potentially dangerous.

File: src/components/TranslateMardown.test.tsx (hunk 1 above)
Comment (vilsbole) [reply src/components/TranslateMardown.test.tsx]: Done

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**